src/main/java/net/dv8tion/jda/api/entities/channel/unions/DefaultGuildChannelUnion.java

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright 2015 Austin Keener, Michael Ritter, Florian Spieß, and the JDA contributors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.dv8tion.jda.api.entities.channel.unions;
+
+import net.dv8tion.jda.api.entities.*;
+
+import javax.annotation.Nonnull;
+
+/**
+ * A specialized union representing all channel types that can be used for the "default" channel for
+ * {@link Guild#getDefaultChannel()} or {@link Member#getDefaultChannel()}.
+ * <br>This class extends {@link StandardGuildChannel} and primarily acts as a discovery tool for
+ * developers to understand which channels might be returned as default channels.
+ *
+ * <br><br><b>Not all {@link StandardGuildChannel} channels can be used as a default channel!</b>
```

Comment:
Instead of `<br><br>`, use `<p>`

Reply:
The `<br><br>` pattern results in more vertical space separation. What's the motivation behind using `<p>` instead?

![image](https://user-images.githubusercontent.com/1479909/174447116-c52bf671-475b-4cdd-a7f2-a2330b50d1a2.png)
![image](https://user-images.githubusercontent.com/1479909/174447136-669d462f-4d1d-4cc6-b803-5d8d9288bc42.png)

Reply:
The primary target for javadoc is the hosted html, which should prefer `<p>` over `<br><br>`. The spacing in intellij is honestly something they should change, since it seems like they treat it like a `<br>` for some reason.

Reply:
Gotcha. Can update.